### ISSUE TITLE
Message for invalid auth token is ugly

### ISSUE THREAD

**Issue by cdavisafc** [cloudfoundry/cli#325]
I'm willing to accept what we do currently as the right thing in the end, but I have to say that a CLI response that looks like this:

auth request failed: Invalid auth token: Invalid refresh token (expired): eyJhbGciOiJSUzI1NiJ9.eyJqdGkiOiJiZDY5OTczMi02ZTY2LTQxNmEtYTE4Ni04YzA2OGE0MzY4YWIiLCJzdWIiOiJiMGYxZjJiZC0zYTExLTRmNDItYWJmNi1hNTllY2UwNjdkNzMiLCJzY29wZSI6WyJzY2ltLnVzZXJpZHMiLCJjbG91ZF9jb250cm9sbGVyLndyaXRlIiwicGFzc3dvcmQud3JpdGUiLCJvcGVuaWQiLCJjbG91ZF9jb250cm9sbGVyLnJlYWQiXSwiaWF0IjoxNDE3NzE0NzI5LCJleHAiOjE0MjAzMDY3MjksImNpZCI6ImNmIiwiaXNzIjoiaHR0cHM6Ly91YWEucnVuLnBpdm90YWwuaW8vb2F1dGgvdG9rZW4iLCJncmFudF90eXBlIjoicGFzc3dvcmQiLCJ1c2VyX25hbWUiOiJjZGF2aXNAZ29waXZvdGFsLmNvbSIsInVzZXJfaWQiOiJiMGYxZjJiZC0zYTExLTRmNDItYWJmNi1hNTllY2UwNjdkNzMiLCJhdWQiOlsic2NpbSIsImNsb3VkX2NvbnRyb2xsZXIiLCJwYXNzd29yZCIsIm9wZW5pZCJdfQ.fOTKzl-EEf4uR0YmddXs-l1t_1jOtuWgoXDqRUG4lzmbV91mt0x8yUxMTIabG688tl-L50X-PWeMH8BWpkMIbgtqhfSJusJN9vpIPoKcc1VWRek_cBLnnAkwZoyc457Dg-1D6DNz6p7V9Mh11tw7cByqTql1TJpzCiVQKAksesgQ4Qf9_ul-ff6DsvUR4LKyiZDosGExlVY-IPxww8U2zBjOCLpqbyua0H24h5Bpq3zUTSA78B2PIm2O6XNEXj67MfjUevMQnFps9J3YDCX4nwRK_qqgXWx3s4EmStffAxCgt4eZiLLK20rzzPxMmvBuUHmlnPq9VYC1XJREyHI3IA expired at Sat Jan 03 17:38:49 UTC 2015

is pretty jarring. All of those characters cannot possibly be helpful. Do we really need to include the token in the response? If this is really used for some type of identification for the token (which seems odd) can we at least do something like:

auth request failed: Invalid auth token: Invalid refresh token (expired): eyJhbGci... expired at Sat Jan 03 17:38:49 UTC 2015

or omit the token entirely:

auth request failed: Invalid auth token: Invalid refresh token Sat Jan 03 17:38:49 UTC 2015

And if someone really needs the token can they just turn tracing on?

**Comment by goehmen** [comment]
@cdavisafc What version of the CLI?  We addressed this issue in 6.7.0 so if you're still seeing it, it's great to find out.  When a token expires now, you get:
```
○ → cf apps
Getting apps in org cli / space hey sup as admin...
FAILED
Authentication has expired.  Please log back in to re-authenticate.

TIP: Use `cf login -a <endpoint> -u <user> -o <org> -s <space>` to log back in and re-authenticate.
```

In 6.7.0 release notes:  
- Improve error message return when refresh token has expired [finishes #78130846]

**Comment by jberkhahn** [comment]
Hey, it looks like this issue was fixed in 6.7.0. We're going to close this issue, but please reopen it if you continue experiencing this issue with a newer version of the CLI.
